.github/scripts/update_schutzfile.py

```diff
@@ -0,0 +1,35 @@
+#!/bin/python3
+
+import json
+import os
+import re
+
+REPO_FILES=os.listdir("/repo/")
+SINGLETONS=[]
+# Get a list of all repositories that contain 'singleton'
+for REPO_FILE in REPO_FILES:
+    with open("/repo/"+REPO_FILE, "r") as f:
+        data = json.load(f)
+    if ( "singleton" in data.keys() ):
+        SINGLETONS.append(data['snapshot-id'])
+
+REPO=os.environ.get('REPO')
+with open(REPO+"/Schutzfile", "r") as f:
+    data = json.load(f)
+
+URLS=[]
+ARCHES=['x86_64', 'aarch64']
+SUFFIX=os.environ.get('SUFFIX')
+# Replace snapshot SUFFIX in repositories that don't have 'singleton' present
+for key in data.keys():
+    if ("repos" in data[key].keys()):
+        for i in range(len(data[key]['repos'])):
+            for ARCH in ARCHES:
+                if any(SINGLETON in data[key]['repos'][i][ARCH][0]['baseurl'] for SINGLETON in SINGLETONS):
+                    continue
+                else:
+                    data[key]['repos'][i][ARCH][0]['baseurl'] = re.sub("[0-9]{8}", SUFFIX, data[key]['repos'][i][ARCH][0]['baseurl'])
+                    print(data[key]['repos'][i][ARCH][0]['baseurl'])
```

Comment:
This can be probably removed.

Reply:
Yes, I probably had it there for debugging.